src/tools/rustdoc-gui/tester.js

```diff
@@ -172,12 +172,7 @@ async function main(argv) {
     files.sort();
 
     console.log(`Running ${files.length} rustdoc-gui tests...`);
-    if (opts["jobs"] < 1) {
-        process.setMaxListeners(files.length + 1);
```

Comment:
We still need this because the default is too low:
```
 (node:17758) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
```

Reply:
You can see it under the section "run the build" on line 9506

Reply:
True, I'll keep them then.